docs/faq.md

```diff
@@ -565,6 +565,17 @@ The Jib build plugins have an extension framework that enables anyone to easily
 
 See the [Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#global-jib-configuration), [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#global-jib-configuration) or [Jib CLI](https://github.com/GoogleContainerTools/jib/blob/master/jib-cli/README.md#global-jib-configuration) docs. Note that the example in the docs uses [Google's Docker Hub mirror on `mirror.gcr.io`](https://cloud.google.com/container-registry/docs/pulling-cached-images).
 
+Starting from Jib build plugins 3.0, [the default base image is `adoptopenjdk` and `jetty` on Docker Hub](default_base_image.md), so you may start to encounter the rate limits if you are not explicitly setting a base image.
+
+Some other alternatives get around the rate limits:
+
+* Prevent Jib from accessing Docker Hub (after Jib cached a base image locally).
```

Comment:
This is super helpful information! Most of these suggestions could be helpful for the CLI as well?

Reply:
Yeah, all of them, except `--offline`, apply to Jib CLI too. The FAQ is a general one, so we should be good.

> See the [Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#global-jib-configuration), [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#global-jib-configuration) or [Jib CLI](https://github.com/GoogleContainerTools/jib/blob/master/jib-cli/README.md#global-jib-configuration) docs.

But I just realized `--offline` is Maven/Gradle-specific, so I'll update the doc to specifically point out that it's only for Jib build plugins.